packages/@ember/application/tests/dependency_injection/normalization_test.js

```diff
@@ -10,7 +10,12 @@ moduleFor(
     constructor() {
       super();
 
-      application = run(Application, 'create');
+      // Must use default resolver because test resolver does not normalize
```

Comment:
Can you update the `moduleFor` name of this to indicate it is specifically for the globals resolver and that it is testing deprecated functionality?

Reply:
Done!